### ISSUE TITLE
[Wishlist] Dirichlet regression

### ISSUE THREAD

**Issue by roaldarbol** [glmmTMB/glmmTMB#1086]
Hi there!
I've been enjoying working with {glmmTMB} a ton. I've been using beta regression quite a bit, which handles cases with two (e.g. proportion time spent active/passive). But with more complex ethograms, containing >2 levels, Dirichlet regression seems to be the way to go (if there are alternative solutions, I'd be happy to be pointed in those directions too). There already exists the [DirichletReg](https://cran.r-project.org/web/packages/DirichletReg/index.html) package, but as far as I can tell they are not mixed models. There's also `dirichlet` family in {brms}, which may be a good place for inspiration. 

I would like to look at ICCs of time budgets (although I'm not aware whether anyone has actually looked at the statistical implementation of this). I commented on an [issue in the {performance} package](https://github.com/easystats/performance/issues/683) for the latter parts. 

Happy to discuss more!

**Comment by bbolker** [comment]
This is a fine wish but the infrastructure of dealing with multivariate responses seems a little bit daunting. Honestly, I would probably start by trying to implement a mixed Dirichlet regression in [RTMB](https://github.com/kaskr/RTMB), although all of the downstream stuff would also be challenging ...

Another alternative (a bit clunky perhaps) would be to do a transformation from the space of compositions to an unconstrained space [e.g. Aitchison/central log ratio/barycentric] and then do a multivariate model by converting to long format (I realize as I write this that it might be completely incomprehensible ...) -- interpretation could be hard though.

**Comment by roaldarbol** [comment]
Yeah, I completely get that. I think initially I'll just use {brms} since they have done it already - I just thought it would be good to have an issue on it here too. I'll have to be honest, I probably won't have the time (or the knowledge!) to attempt implementing it.
